src/metabase/query_processor/util.clj

```diff
@@ -37,6 +37,18 @@
                               :native "native")
                             (codecs/bytes->hex query-hash)))))
 
+(defmulti query->remark
+  "Generate an appropriate REMARK to be prepended to a query to give DBAs additional information about the query being
```

Comment:
Some of our really old code uses all uppercase for parameter names, but for new code you should follow the [Clojure Style Guide](https://guide.clojure.style/) and [use backticks instead](https://guide.clojure.style/#document-pos-arguments).

When we first started writing the Clojure version of Metabase there wasn't really any consensus on highlighting arg names in docstrings yet so I just followed the convention from Emacs Lisp and wrote a bit of Emacs Lisp to highlight them for me. These days pretty much everyone does backticks and my custom Emacs Lisp is long-gone (`clojure-mode`/CIDER highlights backticks automatically now).

My general strategy has been to update the all-uppercase names when I come across them while working on other stuff. Doing it all at once wasn't really appealing. It's been an ongoing process for a few years but I'd say 95% of our code uses backticks now.

So feel free to fix those when you see them if you want. If you don't, it doesn't really make a huge difference to me either. Just make sure not to add any more uppercase param names in new code